src/api/tests/apiKeyService.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { ApiRoleDTO } from '../entities/ApiRole';
+import { validateApiRoles } from '../services/apiKeyService';
+
+describe('#validateApiRoles', () => {
+  test('1 Valid Roles', () => {
```

Comment:
I wonder if we can give this test a more informative name. What do you think about "should return true when the role is valid". And then similar for the other tests.

Reply:
Yeah that makes sense. Have renamed them.